packages/kit/src/routes/config/getStateFromPath.ext.ts

```
@@ -139,74 +152,197 @@ export function getStateFromPath<ParamList extends {}>(
+  const cached = cachedConfigResources.get(options);
+
+  if (cached) return cached;
+
+  const resources = prepareConfigResources(options);
+
+  cachedConfigResources.set(options, resources);
+
+  return resources;
+}
+
+function prepareConfigResources(options?: Options<{}>) {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)